deployments/deploy-ssh.sh

```diff
@@ -482,7 +482,7 @@ spec:
             "--testset-id=$DASH_UID",
             "--branch=$BRANCH",
             "--product=$PRODUCT",
-            "--pds-parameter-json=$PDS_PARAMETER_JSON",
+            "--pds-param-cm=$PDS_PARAMETER_JSON",
```

Comment:
change this variable name also $PDS_PARAMETER_JSON

Reply:
done